src/constants/gl_markers.js

```diff
@@ -17,5 +17,40 @@ module.exports = {
     diamond: '◆',
     'diamond-open': '◇',
     cross: '+',
-    x: '❌'
+    x: '❌',
+    'triangle-up': '▲',
+    'triangle-up-open': '△',
+    'triangle-down': '▼',
+    'triangle-down-open': '▽',
+    'triangle-left': '◄',
+    'triangle-left-open': '◁',
+    'triangle-right': '►',
+    'triangle-right-open': '▷',
+    'triangle-ne': '◥',
+    'triangle-ne-open': '◹',
+    'triangle-nw': '◤',
+    'triangle-nw-open': '◸',
+    'triangle-se': '◢',
+    'triangle-se-open': '◿',
+    'triangle-sw': '◣',
+    'triangle-sw-open': '◺',
+    'pentagon': '⬟',
+    'pentagon-open': '⬠',
+    'hexagon': '⬢',
+    'hexagon-open': '⬡',
+    'hexagon2': '⬣',
+    'star': '★',
+    'star-open': '☆',
+    'hexagram': '✶',
+    'diamond-tall': '♦',
+    'diamond-tall-open': '♢',
+    'hourglass': '⧗',
+    'hourglass-open': '⧖',
+    'bowtie': '⧓',
+    'bowtie-open': '⋈',
+    'circle-cross': '⊕',
+    'circle-x': '⊗',
+    'square-cross': '⊞',
+    'square-x': '⊠',
+    'asterisk': '✱'
```

Comment:
Basically this is going to require passing symbol-dependent offsets and scales to the relevant `gl-vis` modules. @destradafilm  not sure if you want to take a crack at this, I'm afraid it'll be a little bit tedious and @etpinard and I may not be able to get to it for a while.